exercises/practice/wordy/.meta/additional-tests.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "uuid": "88bf4b28-0de3-4883-93c7-db1b14aa806e",
+    "description": "exponential",
+    "comments": [
+      "This test case was added a long time ago.",
+      "Upstreaming it would make the exercise more difficult."
+    ],
+    "property": "exponentials",
+    "input": {
+      "question": "What is 2 raised to the 5th power?"
+    },
+    "expected": 32
+  },
```

Comment:
These cases were in the finished test file, but test generation would remove them, because their not in `canonical-data.json`. I decided to keep them around, but not upstream them.